packages/core/test/actor.test.ts

```diff
@@ -766,18 +790,22 @@ describe('actors', () => {
           states: {
             same: {
               entry: assign({
-                ref: () => spawn(syncChildMachine, falseSyncOption)
+                ref: (_, __, { self, spawn }) =>
+                  spawn(
+                    createMachineBehavior(
+                      syncChildMachine,
+                      self,
```

Comment:
Having to pass `self` here seems like a problem - it should be enforced by the system, not manual because it is actually enforced by the Actor Model (everything forms a tree hierarchy). Not sure how to fix this right now though.

Reply:
This is fixed by using `spawn.from(syncChildMachine)`

Reply:
The reason this isn't done yet is because I still need to augment `createMachineBehavior`'s signature to include options

Reply:
I'm going to keep this as-is because in V5, we want to encourage developers to be explicit about events sent/received between machines. `autoForward` and `sync` are implicit, and furthermore, `autoForward` should be preferred for invoked machines instead.

However, we should keep this advanced option in the tests (not documented) if this behavior is desired.